# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1380771182 14400
#      Wed Oct 02 23:33:02 2013 -0400
# Node ID e7a9be8fdd21703fc9c8df430371fbb4e0f4865d
# Parent  53ae37d37b167d48268cbec305acdcd0dfc393d4
* @ftp/close.m: Fix function name.

diff --git a/scripts/@ftp/close.m b/scripts/@ftp/close.m
--- a/scripts/@ftp/close.m
+++ b/scripts/@ftp/close.m
@@ -18,12 +18,12 @@
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} close (@var{f})
 ## Close the FTP connection represented by the FTP object @var{f}.
 ##
 ## @var{f} is an FTP object returned by the @code{ftp} function.
 ## @end deftypefn
 
-function dir (f)
+function close (f)
   __ftp_close__ (f.curlhandle);
 endfunction
 
